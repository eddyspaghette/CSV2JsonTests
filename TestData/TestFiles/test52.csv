ッ!あ,が,ざ,℁,⏳
ト,ぇ,ご,₣,ψ,😵
ヅ,い,け,₠,Å,○
,ぅ,げ,₡,,☝
デ,,こ,,Δ,⭐
ド,え,さ,℀,ↂ,🍕
ツ,ぃ,ぎ,せ,℟,▓
