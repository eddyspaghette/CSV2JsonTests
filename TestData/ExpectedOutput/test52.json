[
 {
   "ッ!あ": "ト",
   "が": "ぇ",
   "ざ": "ご",
   "℁": "₣",
   "⏳": "ψ",
   "FIELD6": "😵"
 },
 {
   "ッ!あ": "ヅ",
   "が": "い",
   "ざ": "け",
   "℁": "₠",
   "⏳": "Å",
   "FIELD6": "○"
 },
 {
   "ッ!あ": "",
   "が": "ぅ",
   "ざ": "げ",
   "℁": "₡",
   "⏳": "",
   "FIELD6": "☝"
 },
 {
   "ッ!あ": "デ",
   "が": "",
   "ざ": "こ",
   "℁": "",
   "⏳": "Δ",
   "FIELD6": "⭐"
 },
 {
   "ッ!あ": "ド",
   "が": "え",
   "ざ": "さ",
   "℁": "℀",
   "⏳": "ↂ",
   "FIELD6": "🍕"
 },
 {
   "ッ!あ": "ツ",
   "が": "ぃ",
   "ざ": "ぎ",
   "℁": "せ",
   "⏳": "℟",
   "FIELD6": "▓"
 }
]